layers/meta-resin-edison/recipes-core/images/resin-image-initramfs.bbappend

```diff
@@ -0,0 +1 @@
+IMAGE_ROOTFS_MAXSIZE = "16384"
```

Comment:
root@resin:~# fdisk -l /dev/mmcblk0p7                                                                                                                                                                                                        
Disk /dev/mmcblk0p7: 40 MiB, 41943040 bytes, 81920 sectors
Units: sectors of 1 * 512 = 512 bytes
Sector size (logical/physical): 512 bytes / 512 bytes
I/O size (minimum/optimal): 512 bytes / 512 bytes
Disklabel type: dos
Disk identifier: 0x00000000

But:

  root@resin:~# df -h /mnt/boot/
Filesystem      Size  Used Avail Use% Mounted on
/dev/mmcblk0p7  9.1M  8.5M  628K  94% /mnt/boot

So something is off here, but don't think it was itroduced in this PR